extension/css/first-run.css

```diff
@@ -1,12 +1,12 @@
-.first-run {
+.fx-relay-first-run {
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
   height: 100vh;
   padding: 0;
   margin: 0;
-  background-color: linear-gradient(tovar(--grey10), var(--grey20));
+  background-color: linear-gradient(tovar(--relayGrey10), var(--relayGrey20));
```

Comment:
q: is `tovar()` a typo? It isn't new, but I'm not familiar with it (but I'm far from a CSS expert, so I'm delightfully ignorant of this new fancy syntaxes).